src/Playground.res

```diff
@@ -745,7 +738,7 @@ module WarningFlagsWidget = {
     let suggestionBox =
       Option.map(suggestions, elements =>
         <div
-          ref={ReactDOM.Ref.domRef(listboxRef)}
+          ref={ReactDOM.Ref.domRef((Obj.magic(listboxRef): React.ref<Nullable.t<Dom.element>>))}
```

Comment:
`ReactDOM.Ref.domRef` except `React.ref<Nullable.t<Dom.element>>`, but this have type `React.ref<Nullable.t<WebAPI.DOMAPI.htmlElement>>`